build/helpTOC.php

```diff
@@ -172,6 +186,25 @@ protected function doExecute(InputInterface $input, OutputInterface $output): in
         // JSON encode the file and write it to JPATH_ADMINISTRATOR/help/en-GB/toc.json
         file_put_contents(JPATH_ADMINISTRATOR . '/help/en-GB/toc.json', json_encode($toc));
 
+        if (\count($missing)) {
+
+            $str_missing = '';
+
+            foreach ($missing as $string => $value) {
+                $str_missing .= 'COM_ADMIN_HELP_' . $string . '="' . $value . '"'. PHP_EOL;
+            }
+
+            // write missing strings to JPATH_BASE/tmp/missing-helptoc.txt
+            file_put_contents(JPATH_BASE . '/tmp/missing-helptoc.txt', $str_missing);
+
+            $io->comment(sprintf('Number of media-wiki articles without string: %d', \count($missing)));
```

Comment:
Can I suggest changing this so that it is more prominently displayed

```suggestion
            $io->caution(sprintf('Number of media-wiki articles without string: %d', \count($missing)));
```

![image](https://github.com/joomla/joomla-cms/assets/1296369/33c0ac9e-35b5-4da4-b972-229d5f8a5669)

Reply:
Good idea. Thank you!